hsrb_gazebo_bringup/launch/include/hsrb_default_controllers.xml

```diff
@@ -0,0 +1,21 @@
+<launch>
+    <arg name="namespace" default="hsrb" />
+    <arg name="personal_name" default="" />
+
+    <group ns="$(arg personal_name)/$(arg namespace)">
```

Comment:
Why could this namespace not be constructed outside this launch file? Wouldnt we be able to use it as made by toyota then?

Reply:
No, we need this file. As the original contains a tf_prefix.